tests/test_redis.py

```diff
@@ -295,11 +295,11 @@ def testMultitenacyData(self):
         """Test with real data to check if multitenacy is working"""
         p = myredis()
 
-        p1 = "[, 1.0, 0.14, '2014-03-29T19:18:25.784424']"
-        p2 = "[different_server_id, 2.0, 0.48, '2014-03-29T19:18:26.784424']"
+        p1 = "[, 1.0, 0.14, 0.25, 0.25, '2014-03-29T19:18:25.784424']"
```

Comment:
This tests is not good to have the same value in hdd and net, due to we want to check the fact that we obtain the media of the different data. Please change p1 and p2 in order to have different values in hdd and network.